packages/core/directives/map.ts

```diff
@@ -288,7 +293,7 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
     'disableDoubleClickZoom', 'scrollwheel', 'draggable', 'draggableCursor', 'draggingCursor',
     'keyboardShortcuts', 'zoomControl', 'zoomControlOptions', 'styles', 'streetViewControl',
     'streetViewControlOptions', 'zoom', 'mapTypeControl', 'mapTypeControlOptions', 'minZoom',
-    'maxZoom', 'panControl', 'panControlOptions', 'rotateControl', 'rotateControlOptions',
+    'maxZoom', 'controlSize', 'panControl', 'panControlOptions', 'rotateControl', 'rotateControlOptions',
```

Comment:
as per the doc

> This value must be supplied directly when creating the Map, updating this value later may bring the controls into an undefined state. 

So it can't be updated via ngChanges, therefore it shouldn't be in `_mapOptionsAttributes`